testing/jormungandr-automation/resources/explorer/graphql/schema.graphql

```diff
@@ -409,6 +414,11 @@ type Transaction {
 
   """All the blocks this transaction is included in"""
   blocks: [Block!]!
+
+  """
+  Initial bootstrap config params (initial fragments), only present in Block0
+  """
+  initialConfigurationParams: ConfigParams
```

Comment:
I think this should be  `initialConfigurationParams: [ConfigParams]`

Reply:
same here